.vscode/c_cpp_properties.json

```diff
@@ -28,26 +28,42 @@
                     "${workspaceFolder}/demos/",
                     "${workspaceFolder}/doc/",
                     "${workspaceFolder}/effects/",
-                    "${workspaceFolder}/fluid/",
                     "${workspaceFolder}/fonts/",
                     "${workspaceFolder}/fonttools/",
-                    "${workspaceFolder}/manual/",
-                    "${workspaceFolder}/midi/",
                     "${workspaceFolder}/miditools/",
                     "${workspaceFolder}/mscore/",
                     "${workspaceFolder}/mtest/",
                     "${workspaceFolder}/rdoc/",
                     "${workspaceFolder}/share/",
-                    "${workspaceFolder}/synthesizer/",
                     "${workspaceFolder}/test/",
                     "${workspaceFolder}/telemetry/",
                     "${workspaceFolder}/thirdparty/",
                     "${workspaceFolder}/vtest/",
-                    "${workspaceFolder}/zerberus/",
                     "${workspaceFolder}/*"
                 ]
             }
+        },
+        {
+            "name": "Win32",
+            "includePath": [
+                "${workspaceFolder}/**",
+                "C:/Qt/5.12.10/msvc2017_64/bin",
+                "C:/Qt/5.12.10/msvc2017_64/include/**",
+                "C:/Qt/5.12.10/msvc2017_64/include/QtGui"
```

Comment:
So we really need to hardcode these values?

Reply:
Maybe not. These are for not for building anyway. I can just remove them from there and let people choose what they include.

Reply:
Something like `${QT_INSTALL_PREFIX}/`maybe?

Reply:
only available with cmake-initialized environments. I'll remove these include paths for the time being. Individuals will add them if they wish.